server/routes/api/graphql.ts

```diff
@@ -161,13 +160,13 @@ async function findQuestions(
 
 	let items: types.FormItem<Ctx>[] = [];
 	if (user.applied) {
-		items = items.concat(await Promise.all(user.applicationData
+		items = items.concat(await Promise.all(user.applicationData!
 			.reduce(questionFilter, [])
-			.map(item => recordToFormItem(item, user.applicationBranch))
+			.map(item => recordToFormItem(item, user.applicationBranch!))
 		));
 	}
 	if (user.confirmed) {
-		items = items.concat(await Promise.all(user.confirmationData
+		items = items.concat(await Promise.all(user.confirmationData!
 			.reduce(questionFilter, [])
```

Comment:
feel like the `.reduce` and `.map` should be indented another time, since it looks like those are acting on `items` rather than `user.confirmationData`